arlas-core/src/main/java/io/arlas/server/core/model/CollectionReferenceParameters.java

```diff
@@ -47,6 +47,9 @@
     @JsonProperty(value = "centroid_path", required = true)
     public String centroidPath;
 
+    @JsonProperty(value = "h3_path", required = true)
```

Comment:
Having h3 is optional for a collection so the field should also be optional isn't it?

Reply:
ok, done.